### PR TITLE
Trim descriptions of traits

### PR DESCRIPTION
**Pull request by danflapjax**
The descriptions in other types of files have the whitespace trimmed from their ends; this makes the same done for trait files. This should reduce the size of the data JSON file by ~5kB.

### CODE REVIEW

**Comment by StevenClontz** [comment]
Nice! Looks good, will merge assuming tests pass. I'm also going to give you write access to the repo to make branches here directly so tests run automatically.

**Review by Not-Abram — APPROVED** [review]

**Pull request merged**
